styles/globals.css

```diff
@@ -1264,6 +1264,51 @@ div.image .img {
   margin-bottom: 10em;
 }
 
+.compact {
+  padding: .1em;
+  margin-top: -0.25em;
+  margin-left: 1em;
+  margin-right:1em;
+}
+
```

Comment:
Add the following:
```
.nav-button {
    padding: .5em;
}
```

So that the button can align itself with the agenda navbar

Reply:
This should be the final design:
![image](https://github.com/Schrodinger-Hat/osday/assets/6616203/9a526291-5a3c-452c-804c-50b2973e294b)